frontends/election-manager/src/screens/print_test_deck_screen.tsx

```diff
@@ -579,7 +582,7 @@ export function PrintTestDeckScreen(): JSX.Element {
         />
       )}
       {printIndex?.component === 'HandMarkedPaperBallots' &&
-        hmpbTargetElementsCreated && (
+        hmpbTargetElementsCreatedForPrecinctId === currentPrecinctId && (
```

Comment:
Interesting bug!

Because it's now possible to hop straight from one precinct's HMPBs to another's (previously, tally reports and BMD paper ballots always interrupted), it's now also possible for HMPB target elements to exist for the previous precinct but not yet the current precinct. Before this fix, those would get rendered to but then promptly cleared as the `useCreateHmpbTargetElements` hook performed its cleanup, and the print job would print a blank page. We now make sure that HMPB target elements exist for the current precinct before rendering HMPBs.

Planning to add a test to cover this since I only caught this during my manual testing.